Doc/library/inspect.rst

```diff
@@ -515,6 +515,12 @@ attributes (see :ref:`import-mod-attrs` for module attributes):
    :attr:`~method.__func__` attribute (etc) when an object passes
    :func:`ismethod`.
 
+   .. versionchanged:: 3.13
+      For objects with :meth:`~object.__get__` and :meth:`~object.__delete__`
+      but without :meth:`~object.__set__`, ``False`` is now returned
+      (previously, ``True`` was returned for such objects, which was
+      incorrect).
```

Comment:
```suggestion
      Objects with :meth:`~object.__get__` and :meth:`~object.__delete__`
      but without :meth:`~object.__set__` are no more considered as method
      descriptors. In previous versions, they were incorrectly deemed as such.
```

Reply:
@picnixz OK, I committed your suggestion. However, isn't the fragment *are no more considered as method descriptors* too general in its meaning? I mean, someone could misunderstand it – that before Python 3.13 such objects were generally considered as method descriptors; whereas our intent is to inform that just this particular function incorrectly considered it as such.

What do you think about such wording:

> Objects with `__get__()` and `__delete__()` but without `__set__()` are no more considered by this function as method descriptors. In previous versions, they incorrectly were.

Reply:
(Or some other wording that would emphasize the focus on the behavior of this particular function...)

Reply:
I just committed as slightly different variant:

```rest
      Objects with :meth:`~object.__get__` and :meth:`~object.__delete__`
      but without :meth:`~object.__set__` are no more considered by this
      function as method descriptors (in previous versions, they were
      incorrectly deemed as such).
```

Could you please decide whether it is OK. :)

Reply:
I've added a suggestion to tighten this up and make it clear it is describing a bug fix in this function rather than a change to the language itself.

Reply:
Thanks! :)

Reply:
Thank you! :) I applied it.

Reply:
Sure thanks! my suggestions are in the end only... suggestions. I appreciate when someone likes it verbatim but I don't mind when it is reformulated!